src/Widgets/Presets/DefaultSingleitemPreset.php

```diff
@@ -0,0 +1,252 @@
+use Ceres\Widgets\Helper\PresetHelper;
+use Plenty\Modules\ShopBuilder\Contracts\ContentPreset;
+use Plenty\Plugin\Translation\Translator;
+
+class DefaultSingleitemPreset implements ContentPreset
+{
+    /** @var PresetHelper */
+    private $preset;
+
+    /** @var CeresConfig */
+    private $ceresConfig;
+
+    /** @var PresetWidgetFactory */
+    private $twoColumnWidget;
+
+    /** @var PresetWidgetFactory */
+    private $stickyContainer;
+
+    /** @var PresetWidgetFactory */
+    private $tabWidget;
+
+    /** @var Translator */
+    private $translator;
+
+    public function getWidgets()
+    {
+        $this->preset = pluginApp(PresetHelper::class);
+        $this->ceresConfig = pluginApp(CeresConfig::class);
+        $this->translator = pluginApp(Translator::class);
+
+        $this->createTwoColumnWidget();
+        $this->createItemImageWidget();
+        $this->createTabWidget();
+        $this->createStickyContainer();
+        $this->createManufacturer();
+        $this->createNameHeader();
+        $this->createSeparatorWidget();
+        $this->createItemVariationNumber();
+        $this->createItemBundleWidget();
+        $this->createGraduatedPriceWidget();
+        $this->createItemPriceWidget();
+        $this->createItemAvailabilityWidget();
+        $this->createAttributeWidget();
+        $this->createAddToBasketWidget();
+        $this->createAddToWishListWiget();
+        $this->createSeparatorWidget();
+        $this->createLegalInformation();
+
+        return $this->preset->toArray();
+    }
+
+    private function createStickyContainer()
+    {
+        $this->stickyContainer = $this->twoColumnWidget->createChild('second', 'Ceres::StickyContainerWidget');
+    }
+
+    private function createTwoColumnWidget()
+    {
+        $this->twoColumnWidget = $this->preset->createWidget('Ceres::TwoColumnWidget')
+            ->withSetting('layout', 'sevenToFive')
+            ->withSetting('customClass','mt-5');
+    }
+
+    private function createManufacturer()
+    {
+        $this->stickyContainer->createChild('sticky','Ceres::InlineTextWidget')
+            ->withSetting('appearance','none')
+            ->withSetting('customClass','text-muted')
+            ->withSetting('spacing.customPadding', true)
+            ->withSetting('spacing.padding.left.value', 0)
+            ->withSetting('spacing.padding.left.unit', null)
+            ->withSetting('spacing.padding.right.value', 0)
+            ->withSetting('spacing.padding.right.unit', null)
+            ->withSetting('spacing.padding.bottom.value', 0)
+            ->withSetting('spacing.padding.bottom.unit', null)
+            ->withSetting('text',$this->getShopBuilderDataFieldProvider('ManufacturerDataFieldProvider::externalName',array('item.manufacturer.externalName')));
+    }
+    private function createNameHeader()
+    {
+        $dataProvider = $this->getShopBuilderDataFieldProvider('TextsDataFieldProvider::name1',array('texts.name1'));
```

Comment:
HIer sollte eigentlich der Name genommen werden, welcher in der Ceres-Config eingestellt ist

Reply:
So gehts ? aber schön ist anders